src/Raven.Client/Documents/Commands/MultiGet/MultiGetCommand.cs

```diff
@@ -166,44 +167,41 @@ private string GetCacheKey(GetRequest command, out string requestUrl)
         {
-                var state = new JsonParserState();
-                using (var parser = new UnmanagedJsonParser(context, state, "multi_get/response"))
+            var state = new JsonParserState();
+            using (var parser = new UnmanagedJsonParser(context, state, "multi_get/response"))
             using (context.GetMemoryBuffer(out var buffer))
-                using (var peepingTomStream = new PeepingTomStream(stream, context))
+            using (var peepingTomStream = new PeepingTomStream(stream, context))
             using (_cached)
-                {
-                    if (UnmanagedJsonParserHelper.Read(peepingTomStream, parser, state, buffer) == false)
-                        ThrowInvalidJsonResponse(peepingTomStream);
+            {
+                if (UnmanagedJsonParserHelper.Read(peepingTomStream, parser, state, buffer) == false)
+                    ThrowInvalidJsonResponse(peepingTomStream);
 
-                    if (state.CurrentTokenType != JsonParserToken.StartObject)
-                        ThrowInvalidJsonResponse(peepingTomStream);
+                if (state.CurrentTokenType != JsonParserToken.StartObject) 
+                    ThrowInvalidJsonResponse(peepingTomStream);
 
-                    var property = UnmanagedJsonParserHelper.ReadString(context, peepingTomStream, parser, state, buffer);
-                    if (property != nameof(BlittableArrayResult.Results))
-                        ThrowInvalidJsonResponse(peepingTomStream);
+                var property = UnmanagedJsonParserHelper.ReadString(context, peepingTomStream, parser, state, buffer);
+                if (property != nameof(BlittableArrayResult.Results))
+                    ThrowInvalidJsonResponse(peepingTomStream);
 
-                    var i = 0;
+                var i = 0;
                 Result = new List<GetResponse>(_commands.Count);
-                    foreach (var getResponse in ReadResponses(context, peepingTomStream, parser, state, buffer))
-                    {
-                        var command = _commands[i];
-
-                        MaybeSetCache(getResponse, command);
+                foreach (var getResponse in ReadResponses(context, peepingTomStream, parser, state, buffer))
+                {
+                    var command = _commands[i];
 
-                    Result.Add(_cached != null && getResponse.StatusCode == HttpStatusCode.NotModified
-                        ? new GetResponse { Result = _cached.Values[i].Cached?.Clone(context), StatusCode = HttpStatusCode.NotModified }
-                        : getResponse);
+                    MaybeSetCache(getResponse, command);
+                    Result.Add(_cached != null && getResponse.StatusCode == HttpStatusCode.NotModified ? new GetResponse { Result = _cached.Values[i].Cached?.Clone(context), StatusCode = HttpStatusCode.NotModified } : getResponse);
```

Comment:
Let's try to avoid those kind of changes in the future. It is hard to figure out if something is going on.

Reply:
Ok